### ISSUE TITLE
Some means of handling hard drive reporting when number of drives exceeds screen depth

### ISSUE THREAD

**Issue by DefueBugs** [Zren/plasma-applet-sysmonitordash#10]
The current dashboard screen display does not appear to have scrollable columns. This especially a problem when the number of hard drives exceeds the maximum number of drives which can be displayed in the current screen depth.

**Comment by Zren** [comment]
Yeah, it needs a ScrollView. I'd need to devise a way to close the Dashboard (Esc + close button probably) as you'd need to support scrolling using the mouse/finger. We need a margin around the screen edge anyways so we could stick a few close buttons in it.

**Comment by Zren** [comment]
I made each column scrollable, but made sure that the CPU/RAM/Swap graphs are always visible in the center column. Do you think that looks odd that just the networks are scrollable? Should I just make the entire center column scroll?

![2019-03-03___01-35-30](https://user-images.githubusercontent.com/416367/53691931-7d95e200-3d55-11e9-87f6-11fa4e487606.png)
